corehq/motech/tests/test_forms.py

```diff
@@ -0,0 +1,41 @@
+from django.test import SimpleTestCase
+
+from ..forms import ConnectionSettingsForm, UnrecognizedHost
+
+from corehq.util.urlsanitize.test.mockipinfo import hostname_resolving_to_ips, unresolvable_hostname
+
+import time
```

Comment:
Nit: Please use `isort`

Reply:
Thanks for pointing this out, I'll configure that

Reply:
This is very cool. Are there situations where using `isort` is undesirable? I'm tempted to automate this through a pre-commit hook